### ISSUE TITLE
ResourceOwnerPasswordValidator in AccountController

### ISSUE THREAD

**Issue by ebarbuti** [IdentityServer/IdentityServer4#435]
We currently have a custom database for user storage and we are moving to implement IdentityServer4 as soon as it is ready!  I have a project up and running and it works great using Postman to test etc.  I grabbed the UI from one of the Quickstarts but its using the InMemoryUserLoginService etc in the AccountController.  Is there an example of an AccountController that uses RecourceOwnerPasswordValidator? Is that even what I should be looking for? So far so good from the server project though... keep up the good work and sorry if this is a simple one .. I am definately a newbie!

**Comment by brockallen** [comment]
There's a different interface for that flow: https://identityserver4.readthedocs.io/en/dev/topics/resource_owner.html

**Comment by ebarbuti** [comment]
Thanks for the quick reply.. yes.. I have already implemented that and have it set up in startup.cs

services.AddTransient<IResourceOwnerPasswordValidator, Configurations.ResourceOwnerPasswordValidator>();
services.AddTransient<IProfileService, Configurations.ProfileService>();

This works great from Postman but not in the UI I grabbed from one of the quickstarts you provided.
The AccountController does not like InMemoryUserLoginService... so I "think" I need to use the ResourceOwnerPasswordValidator from there too.. maybe I'm confused.. well wait.. yes.. I am confused!
Is this the correct approach from within my AccountController?
